src/main/java/genie/task/Task.java

```diff
@@ -28,5 +31,16 @@ public String getStatusBox() {
     public String toFileFormat() {
         return this.getStatusBox() + this.description;
     }
+    public boolean contains(String keyword) {
+        return description.contains(keyword);
+    }
+    public String getDescription() {
+        return description;
+    }
+    public boolean containsWord(String keyword) {
+        Pattern pattern = Pattern.compile("\\b" + keyword + "\\b", Pattern.CASE_INSENSITIVE);
```

Comment:
Love that you are using a regex pattern to make Task "recognize" words on their own!